ecl/hthor/hthor.cpp

```diff
@@ -372,13 +375,19 @@ ClusterWriteHandler *createClusterWriteHandler(IAgentContext &agent, IHThorIndex
         }
         if (!cluster)
             break;
-        if(!clusterHandler)
+        if (!clusterHandler)
         {
-            if(extend)
-                throw MakeStringException(0, "Cannot combine EXTEND and CLUSTER flags on disk write of file %s", lfn);
+            if (extend)
+                throw makeStringExceptionV(ENGINEERR_EXTEND_CLUSTER_WRITE, "Cannot combine EXTEND and CLUSTER flags on disk write of file %s", lfn);
             clusterHandler.setown(new CHThorClusterWriteHandler(lfn, "OUTPUT", agent));
         }
         clusterHandler->addCluster(cluster);
+        Owned<IPropertyTree> plane = getStoragePlane(cluster);
+        bool thisPlaneCompressed = plane ? plane->getPropBool("@compressLogicalFiles") : false;
```

Comment:
Add a comment to indicate this is checking the consistency, rather than determining whether it should be compressed.  (I wondered why it wasn't using a default.)

Reply:
will add comment